tests/store_helpers.py

```diff
@@ -386,38 +391,42 @@ def ensure_case_from_dict(
     ):
         """Load a case with samples and link relations"""
         customer_obj = self.ensure_customer(store)
-        case_obj = store.Family(
-            name=case_info["name"],
-            panels=case_info["panels"],
-            internal_id=case_info["internal_id"],
-            ordered_at=ordered_at,
-            data_analysis=case_info.get("data_analysis", str(Pipeline.MIP_DNA)),
-            data_delivery=case_info.get("data_delivery", str(DataDelivery.SCOUT)),
-            created_at=created_at,
-            action=case_info.get("action"),
-        )
-
-        case_obj = self.add_case(store, case_obj=case_obj, customer_id=customer_obj.internal_id)
 
         app_tag = app_tag or "WGSPCFC030"
         app_type = case_info.get("application_type", "wgs")
         self.ensure_application_version(store, application_tag=app_tag)
 
+        ticket: Optional[int] = None
         sample_objs = {}
         for sample_data in case_info["samples"]:
             sample_id = sample_data["internal_id"]
+            ticket = sample_data["ticket_number"]
             sample_obj = self.add_sample(
                 store,
+                application_type=app_type,
+                capture_kit=sample_data["capture_kit"],
                 gender=sample_data["sex"],
-                name=sample_data.get("name"),
                 internal_id=sample_id,
-                application_type=app_type,
-                ticket=sample_data["ticket_number"],
+                name=sample_data.get("name"),
                 reads=sample_data["reads"],
-                capture_kit=sample_data["capture_kit"],
+                ticket=ticket,
```

Comment:
Is ticket still present on each sample? Is it the first ticket or the last? Or should it be a list?

Reply:
Ticket would still be present of sample and it is the ticket in which the sample was first ordered